src/services/apis.ts

```diff
@@ -18,6 +18,28 @@ function getAccountInformation<T>(params: any): NetworkPromiseResponse<T> {
   });
 }
 
+/**
+ *  Fetch accounts information by passing public keys.
+ * `NetworkPromiseResponse<T>`
+ * @param {any} params - any
+ * @returns A promise.
+ */
+function getAccounts<T>(params: any): NetworkPromiseResponse<T> {
+  return new Promise((resolve, reject) => {
+    network
+      .authorizedRequest('users/', 'POST', params)
```

Comment:
Please update this api for testnet. Api reponse: `url: '/users/ not found'`

Reply:
I updated.